#1893 autoclean on s3 (#1924)
AbsaOSS/enceladus: _docs/2.0.0/usage/config.md

<!-- specific sections on Standardization & Conformance options may follow in the future -->

Note, that

>   - `conformance.autoclean.standardized.hdfs.folder` when set to true and the job is writing to S3, there will be a leftover empty file like `conformance-output_$folder$` after the autoclean.

Reviewer: I would like to have this note linked from the `conformance.autoclean.standardized.hdfs.folder`, similarly as you can see in https://absaoss.github.io/enceladus/2021/02/12/release-2.18.1.html

Reviewer: ```suggestion
  - `conformance.autoclean.standardized.hdfs.folder` when set to true and the job is writing to S3, there could be a leftover empty file like `conformance-output_$folder$` after the autoclean.
```
The file is put there only in case the folder would be all empty. If there are other folders, it won't be created.

>   This, however, will not negatively impact the functionality of other jobs and is due to the EMR comitter. The recommendation is that the same `standardized.hdfs.path` value to be set for

Reviewer: ```suggestion
  This, however, will not negatively impact the functionality of other jobs even when using the same path and is due to the EMR comitter.
```
Here I would end the _Note_, the rest is S3/AWS thing not Enceladus.
